src/main/java/com/example/demo/dto/CreateBookRequestDto.java

```diff
@@ -0,0 +1,14 @@
+package com.example.demo.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class CreateBookRequestDto {
+    private String title;
```

Comment:
```suggestion
    @NotBlank
    private String title;
```

Reply:
Add validation for all necessary fields

Reply:
Column - is for entity classes
We do not need validation for now. We will have a separate task to add it.